src/view/xwayland.cpp

```diff
@@ -178,7 +178,7 @@ class wayfire_xwayland_view_base : public wf::wlr_view_t
                 &output_geometry_changed);
         }
         /* Update the real position */
-        send_configure();
+        send_configure(XCB_CONFIG_WINDOW_X | XCB_CONFIG_WINDOW_Y | XCB_CONFIG_WINDOW_WIDTH | XCB_CONFIG_WINDOW_HEIGHT);
```

Comment:
why do we need width and height here? It is again position-update-only

Reply:
Without this, xterm ends up starting with 3x3 size.